frontend/lib/api/form/pollingstation/PollingStationFormController.tsx

```diff
@@ -71,16 +74,21 @@ export type AnyFormReference =
   | FormReferencePoliticalGroupVotes
   | FormReferenceSave;
 
+interface SubmitCurrentFormOptions {
+  acceptWarnings?: boolean;
```

Comment:
This is not part of values yet?
would make storing clientState easier?

Reply:
The backend doesn't use this, so that's why it's part of the client state.

We could return it from the form component in a different way as part of #402.